zalando-incubator/zally
# 222 Add Release Script to Gradle

Closes #222 

🔋 

### How to test
1. Run `./gradlew clean build -Pversion=1.0`
2. Observe release is created in `releases/` folder
3. Observe `bin/zally` is still working and `bin/zally.jar` is up to date
